vendor/ember-cli-deprecation-workflow/main.js

```diff
@@ -1,11 +1,11 @@
-/* globals self */
+/* globals require Ember */
```

Comment:
undo all changes in this file for prettier. This is not needed as it will be deleted shortly. reset it back to master/main and use eslint-disable to ignore warnings 👍